holidays/countries/russia.py

```diff
@@ -120,4 +120,13 @@ class RussiaStaticHolidays:
             (FEB, 24, tr("День защитника Отечества")),
             (MAY, 8, tr("День Победы")),
         ),
+        # Postponed days 2024
+        # src: https://www.consultant.ru/document/cons_doc_LAW_455140/
+        2024: (
+            (APR, 29, tr("Перенос 27.04.2024")),
+            (APR, 30, tr("Перенос 02.11.2024")),
+            (MAY, 10, tr("Перенос 06.01.2024")),
+            (DEC, 30, tr("Перенос 28.12.2024")),
+            (DEC, 31, tr("Перенос 07.01.2024")),
```

Comment:
```suggestion
            (APR, 29, APR, 27),
            (APR, 30, FEB, 11),
            (MAY, 10, JUN, 1),
            (DEC, 30, DEC, 28),
            (DEC, 31, JAN, 7),
```

Reply:
Make sure to include the following section below Line 115, too (after the `class RussiaStaticHolidays:` line)

```python3
        # Date format (see strftime() Format Codes).
        substituted_date_format = tr("%d.%m.%Y")
        # Postpone of %s.
        substituted_label = tr("Перенос %s")
```